insta/src/macros.rs

```diff
@@ -321,7 +319,7 @@ macro_rules! assert_debug_snapshot {
 // the value. This allows us to implement other macros with a small wrapper. All
 // snapshot macros eventually call this macro.
 //
-// The macro handles optional trailing commas in file snapshots.
+// This macro is expected to handle trailing commas.
```

Comment:
```suggestion
// This macro handles trailing commas.
```

Reply:
Like this? db010fb8439d8dc43e31b644cb9db96a9e958678

Reply:
Yes cool! (FYI GH also allows "Accept suggestion")